pallets/aleph/src/impls.rs

```diff
@@ -49,4 +53,15 @@ where
             }
         }
     }
+
+    fn update_next_session_finality_version() {
```

Comment:
that looks like some sort of fix? If so should this should halt the abft version upgrade in the testnet probably

Reply:
It basically check whether there is a `VersionChange` on the next session and, if so, it swaps out the `NextSessionFinalityVersion` `StorageItem` with the version for the next session. By including this, we do not have to have a separate RPC call for the `NextSessionFinalityVersion`.